modules/programs/i3status.nix

```
@@ -0,0 +1,107 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.i3status;
+
+  toStr = val:
+    if isString val then ''"${val}"''
+    else if isBool val then (if val then "true" else "false")
+    else toString val;
+
+  blockToConfig = block: ''${block.type} ${optionalString (block.name != null) ''"${block.name}"''} {
```

Can split the line to make it a bit shorter:
```suggestion
  blockToConfig = block: ''
    ${block.type} ${optionalString (block.name != null) ''"${block.name}"''} {
```